server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -192,14 +198,16 @@ export default class BulkChangeModal extends React.PureComponent {
         this.setState({ submitting: false });
         this.props.onDone();
       },
-      (error: Error) => {
+      (error /*: Error */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)